.github/workflows/reusable-audit.yml

```
@@ -29,13 +29,9 @@ jobs:
           node-version-file: .nvmrc
       - name: Install Node.js dependencies
         run: npm ci
-      - name: Audit all npm dependencies
+      - name: Audit npm dependencies
         if: ${{ !startsWith(matrix.ref, 'v') }}
         run: npm run audit
```

Without an alternative for if `matrix.ref` starts with a 'v':

```suggestion
      - name: Audit npm dependencies
        run: npm run audit
```